scrapper.py

```diff
@@ -1,6 +1,16 @@
 """
 Scrapper implementation
 """
+import json
+import os
+
+import requests
+from bs4 import BeautifulSoup
+from random_user_agent.user_agent import UserAgent
+from random_user_agent.params import SoftwareName, OperatingSystem
+from constants import ASSETS_PATH
+from constants import CRAWLER_CONFIG_PATH
```

Comment:
you can import it all at once
`from constants import ASSETS_PATH, CRAWLER_CONFIG_PATH`

Reply:
also, it is better to import everything in alphabetical order

Reply:
done